graveler/committed/meta_range_writer_test.go

```diff
@@ -118,10 +118,33 @@ func TestWriter_RecordRangeAndClose(t *testing.T) {
 	mockMetaWriter.EXPECT().GetApproximateSize().Return(uint64(0)).AnyTimes()
 
 	// write two records on MetaRange and one for Range
-	mockWriter.EXPECT().WriteRecord(gomock.Any())
-	mockMetaWriter.EXPECT().WriteRecord(gomock.Any()).Times(2)
-	mockWriter.EXPECT().Close().Return(&committed.WriteResult{}, nil)
-	mockMetaWriter.EXPECT().Close().Return(&committed.WriteResult{}, nil)
+	call0 := mockWriter.EXPECT().WriteRecord(gomock.Any())
+	call1 := mockWriter.EXPECT().Close().Return(&committed.WriteResult{
+		RangeID: "rng-id",
+		First:   []byte("a"),
+		Last:    []byte("a"),
+		Count:   1,
+	}, nil).After(call0)
+	call2 := mockMetaWriter.EXPECT().WriteRecord(getExpected(t, graveler.ValueRecord{
+		Key: []byte("a"),
+		Value: &graveler.Value{
+			Identity: []byte("rng-id"),
+			Data: mustMarshalRange(committed.Range{
+				ID:     "rng-id",
+				MinKey: []byte("a"),
+				MaxKey: []byte("a"),
+				Count:  1,
+			}),
+		},
+	})).After(call1)
```

Comment:
This is over-specified.  It would be correct to perform this write at any time.  If you want to sequence things, then sequence the `GetWriter` calls to return different mock writers, and verify that each mock gets the right writes.